Userland/Libraries/LibGLSL/Tests/parser/discard.glsl

```diff
@@ -0,0 +1,4 @@
+void foo()
+{
+    discard;
+}
```

Comment:
You seem to be quite inconsistent, if test files should have a trailing new line
If it does not affect the AST it is better to have one, and a lot of editors will append one on safe, needlessly changing the file